app/models/persistent_volume.rb

```diff
@@ -1,4 +1,15 @@
 class PersistentVolume < ContainerVolume
   acts_as_miq_taggable
+  serialize :capacity
```

Comment:
and as you are going with native arrays in psql, how about using hstore here?
http://edgeguides.rubyonrails.org/active_record_postgresql.html#hstore

@carbonin @Fryguy whats our opinion on native psql extensions?